fideslog/sdk/python/client.py

```diff
@@ -1,6 +1,8 @@
 # pylint: disable=too-many-arguments
 
-from asyncio import run
+import asyncio
+import sys
+from asyncio import run, set_event_loop_policy
```

Comment:
`WindowsSelectorEventLoopPolicy` is not importable on macOS. Also, let's keep these as organized and minimal as possible.
```suggestion
from asyncio import SelectorEventLoop, run, set_event_loop
from selectors import SelectSelector
from sys import platform, version_info
```

Reply:
add `isort`?